src/js/apps/patients/schedule/reduced-schedule/reduced_schedule_app.js

```diff
@@ -2,18 +2,60 @@ import Radio from 'backbone.radio';
 
 import App from 'js/base/app';
 
+import StateModel from './reduced_schedule_state';
+
+import FiltersApp from 'js/apps/patients/schedule/filters_app';
```

Comment:
I've split the `reduced_schedule_app.js` and `reduced_schedule_state.js` into their own directory (`/patients/schedule/reduced-schedule`).

But I'm still using the same `FiltersApp` shared by the schedule. I didn't want to create a new `reduced_filters_app` because it'd be identical.